### ISSUE TITLE
[Question] Simple subvol pull/push

### ISSUE THREAD

**Issue by Magissia** [digint/btrbk#346]
Is it possible with btrbk to just pull/push subvolumes containing other subvolumes without renaming them and without taking care of any automatique cleanup, only checking to not pull/push the same subsubvolume twice ?

**Comment by digint** [comment]
No, there is no automatic inclusion of sub-subvolumes. what you can do is use wildcards in your `subvolume` config (e.g. `subvolume vol0/*`)
